src/controllers/auth.js

```diff
@@ -0,0 +1,25 @@
+import { StatusCodes } from "http-status-codes";
+import bcrypt from "bcryptjs";
+import { findUserByEmail } from "../services/auth/auth.service";
+import { generateToken } from "../utils/auth";
+import { formatResponse } from "../utils/format";
+
+export const login = async (req, res) => {
+  const { email, password } = req.body;
+  const user = await findUserByEmail(email);
+
+  if (!user)
+    return formatResponse(res, StatusCodes.NOT_FOUND, null, "User not found");
```

Comment:
This is fine. But it might give intruders a way to know if an email is registered or not.
That's why normally on the login form you get a message saying Invalid credentials wether the user exists or not and/or the password is incorrect